core/src/zeit/push/tests/test_urbanairship.py

```diff
@@ -246,6 +241,78 @@ def test_publish_triggers_push_notification_via_message_config(self):
         self.assertEqual(calls[0][2].get('type'), 'mobile')
 
 
+class ChannelsTest(zeit.push.testing.TestCase):
+    def setUp(self):
+        super().setUp()
+        content = ExampleContentType()
+        content.title = 'content_title'
+        self.repository['content'] = content
+        self.content = self.repository['content']
+        self.create_payload_template('{"messages": {"one": 1}}', 'coffee.json')
+        template = self.repository['data']['urbanairship-templates']['coffee.json']
+        with zeit.cms.checkout.helper.checked_out(template) as co:
+            co.channels = (('Push', 'coffee'),)
+        self.create_payload_template('{"messages": {"two": 2}}', 'cake.json')
+        template = self.repository['data']['urbanairship-templates']['cake.json']
+        with zeit.cms.checkout.helper.checked_out(template) as co:
+            co.channels = (('Push', 'cake'),)
+        self.create_payload_template('{"messages": {"three": 3}}', 'cookies.json')
+        template = self.repository['data']['urbanairship-templates']['cookies.json']
+
+    def message_config(self, name):
+        return {
+            'enabled': 1,
```

Comment:
```suggestion
            'enabled': True,
```
wäre die "realistische" Nutzung